scipy/optimize/tests/test_zeros.py

```diff
@@ -836,3 +836,15 @@ def f(x):
     res = res if rs_interface else res[1]
     assert res.converged
     assert_allclose(res.root, 0, atol=1e-8)
+
+
+def test_newton_complex_gh10103():
+    # gh-10103 reported a problem with `newton` and complex x0.
```

Comment:
Perhaps expand the description to something like:
```
# gh-10103 reported a problem with `newton` and complex x0, no fprime and no x1.
```

Reply:
Thanks. Done!